os/kernel/clock/clock_gettime.c

```diff
@@ -116,6 +116,9 @@ int clock_gettime(clockid_t clock_id, struct timespec *tp)
 
 	svdbg("clock_id=%d\n", clock_id);
 	DEBUGASSERT(tp != NULL);
+	if (!tp) {
```

Comment:
Much minor comment.
You check the variable before svdbg for clock_id at previous function.
Let's do that with same sequence.